src/commands/index.ts

```diff
@@ -228,6 +229,8 @@ const commands: CommandDefinition[] = [
     salty,
     airac,
     mico,
+    salty,
```

Comment:
There is an extra `salty,` and `headwind,` is out of line. with the imports list

Would suggest checking you are in line and moving your imports to the end of the list